src/AceEditor/madie-ace-editor.tsx

```diff
@@ -198,4 +214,38 @@ const MadieAceEditor = ({
   );
 };
 
+// const CheckLogin = async (): Promise<Boolean> => {
+//   const terminologyServiceApi = useTerminologyServiceApi();
+//   let isLoggedIn = false;
+//   await terminologyServiceApi
+//     .checkLogin()
+//     .then(() => {
+//       isLoggedIn = true;
+//     })
+//     .catch((err) => {
+//       isLoggedIn = false;
+//     });
+//   return isLoggedIn;
+// };
+
+export const useTranslateCqlToElm = async (
+  cql: string
+): Promise<ElmTranslation> => {
+  return await useTranslateCql(cql);
+};
+
+export const useValidateCodes = async (
+  customCqlCodes: CustomCqlCode[],
+  loggedInUMLS: boolean
+): Promise<ElmTranslationError[]> => {
+  return await useValidateCustomeCqlCodes(customCqlCodes, loggedInUMLS);
+};
+
+export const validateValueSets = async (
+  valuesetsArray: ElmValueSet[],
+  loggedInUMLS: boolean
+): Promise<ElmTranslationError[]> => {
+  return await getValueSetErrors(valuesetsArray, loggedInUMLS);
```

Comment:
await is not required, Since a promise is being returned. Applies for all 3 functions.